scripts/e2e-test.sh

```diff
@@ -30,6 +31,7 @@ Options:
                             prior to uninstall.
   --onfail <stop|continue>  On fail, stop immediately or continue default($ON_FAIL)
                             Behaviour for "continue" only differs if uninstall is in the list of tests (the default).
+  --uninstall_cleanup <x>   On uninstall cleanup for reusable cluster, "0" disables this behaviour. default($CLEANUP)
```

Comment:
What is the purpose of an argument? Bool flags are usually constructed w/o argument as:
```
--uninstall-cleanup
--no-uninstall-cleanup
```

I just realized that `--onfail` is essentialy also bool flag in disguise :)

Reply:
The objective of using the argument is to select a behaviour independent of the default setting.
If specified a particular selection has been specified in the command line (of a.n.other script) that will hold true.